features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -404,6 +404,13 @@
     <configfile finalname="${openhab.conf}/services/mqtt-eventbus.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/mqtt-eventbus</configfile>
   </feature>
 
+  <feature name="openhab-binding-mqttitude" description="Mqttitude (Owntracks) Binding" version="${project.version}">
+    <feature>openhab-runtime-base</feature>
+    <feature>openhab-runtime-compat1x</feature>
```

Comment:
Should probably do same as MQTT item binding by adding the line to start the MQTT transport bundle here.